subgraphs/_eslint-rules/no-string-literals.js

```diff
@@ -0,0 +1,92 @@
+module.exports = {
+  meta: {
+    docs: {
+      description:
+        "string literals is only allowed in log argument or in UPPER_CASE variable in constants.ts or in logs",
+      category: "Stylistic Issues",
+      recommended: true,
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      forbidden:
+        "String literal must be in log argument or UPPER_CASE variable",
+    },
+  },
+
+  create: function (context) {
+    function checkLogCallOrStringMethods(node, ancestors, logValue) {
+      const callExpressionAncestor = ancestors.find(
+        (ancestor) => ancestor.type === "CallExpression"
+      );
+
+      if (callExpressionAncestor) {
+        const callee = callExpressionAncestor.callee;
+        if (
+          callee.type === "MemberExpression" &&
+          (callee.object.name === "log" ||
+            // these may need to be expanded to include more methods
+            ["fromString"].includes(callee.property.name) ||
+            ["concat", "toLowerCase"].includes(callee.property.name))
```

Comment:
I don't think `.toLowerCase()` is necessary here.

You could add `fromHexString()` as well

Reply:
done.